src/Core/ViewHelper/Traits/CompileWithContentArgumentAndRenderStatic.php

```diff
@@ -25,12 +25,34 @@ trait CompileWithContentArgumentAndRenderStatic
      * registered in `initializeArguments` of the ViewHelper
      * will be used.
      *
-     * Note: it is significantly better practice to define
+     * Note: it is significantly better practice defining
      * this property in your ViewHelper class and so fix it
      * to one particular argument instead of resolving,
      * especially when your ViewHelper is called multiple
      * times within an uncompiled template!
      *
+     * This property cannot be directly set in consuming
+     * ViewHelper, instead set the property in ViewHelper
+     * constructor '__construct()', for example with
+     * $this->contentArgumentName = 'explicitlyToUseArgumentName';
+     *
+     * Another possible way would be to override the method
+     * 'resolveContentArgumentName()' and returning the name.
+     *
+     * public function resolveContentArgumentName()
+     * {
+     *     return 'explicitlyToUseArgumentName';
+     * }
+     *
+     * Note: Setting this through 'initializeArguments()' do
+     *       not work as it may be expected, and the other
+     *       methods should be avoided to override for this.
+     *
+     * Following test ViewHelpers are tested and demonstrates
+     * that the setting posibillities works.
+     * @see CompileWithContentArgumentAndRenderStaticExplicitSetArgumentNameForContentInConstructorViewHelper
+     * @see CompileWithContentArgumentAndRenderStaticExplicitSetArgumentNameForContentOverriddenResolveContentArgumentNameMethodViewHelper
```

Comment:
This should be dropped, since it doesn't add value to the consumer and will get outdated over time.